sklearn/manifold/spectral_embedding_.py

```diff
@@ -320,7 +321,7 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
             # doesn't behave well in low dimension
             X = random_state.rand(laplacian.shape[0], n_components + 1)
             X[:, 0] = dd.ravel()
-            lambdas, diffusion_map = lobpcg(laplacian, X, tol=1e-15,
+            lambdas, diffusion_map = lobpcg(laplacian, X, tol=1e-5,
```

Comment:
Ditto

Reply:
Tested this and it appears it's not necessary. Reverting them back now.

Reply:
Actually, after testing many different seeds, the increased tolerances are necessary...it only becomes more rare to see them as the tolerance is increased. 1e-5 is a safe tolerance, at least for the hundreds of seeds I have tested.

Reply:
Could the codecov failure be because this else is no longer run as a result of the change in tolerance above??

Reply:
I've not looked at the codecov report to see which lines are missed

Reply:
TBH, I have no idea what this test is for or how to interpret the report.

Reply:
Yes, this section is no longer run in tests. Any chance you can come up with code that runs this else case? It seems to be a result of the change in tolerance above, since your pr passed codecov when you had reverted the tolerance changes